test/conictest.jl

```diff
@@ -3,130 +3,112 @@
 #  License, v. 2.0. If a copy of the MPL was not distributed with this
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-function runconictests(algorithm, mip_solvers, conic_solvers)
-
-facts("Default solvers test") do
-    x = Convex.Variable(1,:Int)
-
-    problem = Convex.maximize( 3x,
-                        x <= 10,
-                        x^2 <= 9)
-    
-    Convex.solve!(problem, PajaritoSolver())
-    @fact problem.optval --> roughly(9.0, TOL)
-end
-
-facts("Default MIP solver test") do
-    for conic_solver in conic_solvers
-context("With $algorithm, $(typeof(conic_solver))") do
-        x = Convex.Variable(1,:Int)
-
-        problem = Convex.maximize( 3x,
-                            x <= 10,
-                            x^2 <= 9)
-        
-        Convex.solve!(problem, PajaritoSolver(cont_solver=conic_solver))
-        @fact problem.optval --> roughly(9.0, TOL)
-end
-    end
-end
-
-facts("Default DCP solver test") do
-    for mip_solver in mip_solvers
-context("With $algorithm, $(typeof(mip_solver))") do
-        x = Convex.Variable(1,:Int)
-
-        problem = Convex.maximize( 3x,
-                            x <= 10,
-                            x^2 <= 9)
-        
-        Convex.solve!(problem, PajaritoSolver(mip_solver=mip_solver))
-        @fact problem.optval --> roughly(9.0, TOL)
-end
-    end
-end
-
-facts("Infeasible Conic problem") do
-
-    for mip_solver in mip_solvers
-        for conic_solver in conic_solvers
-context("With $algorithm, $(typeof(mip_solver)) and $(typeof(conic_solver))") do
+function runconictests(branch_cut, mip_solver, conic_solver)
+    algorithm = branch_cut ? "BC" : "OA"
+
+    # facts("Default solvers test") do
```

Comment:
to avoid redundancy, in new commit we test the default mip and conic solvers once